controller/search.go

```diff
@@ -69,10 +69,6 @@ func (a WorkItemPtrSlice) Swap(i, j int) {
 	a[i], a[j] = a[j], a[i]
 }
 
-// Ensure WorkItemPtrSlice implements the sort.Interface
-var _ sort.Interface = WorkItemPtrSlice{}
```

Comment:
If `WorkItemPtrSlice` no more implementing `sort.Interface`. You can remove the implementation details (`Len`, `Less`, and `Swap` methods) also.

Reply:
May be `WorkItemPtrSlice` itself is no more required?

Reply:
removed it here 579928f